yg-pe.py

```diff
@@ -43,14 +42,15 @@ def create_my_orders(self):
             # minimum size bumped if necessary such that you always profit
             # least 50% of the miner fee
             self.minsize = max(
-                int(1.5 * self.txfee / float(self.cjfee_r)), self.minsize)
+                int(1.5 * self.txfee / float(self.cjfee_r)), max_minsize)
```

Comment:
I changed the name because `self.minsize` and `minsize` had the same name and I believed that made it less readable.

It shouldn't make a difference, `self.minsize` was either set to `minsize` (which is now called max_minsize) or to the `int(1.5 * self.txfee / float(self.cjfee_r))` expression which is the same every time its evaluated.